src/system-health/scripts/healthd

```diff
@@ -5,6 +5,9 @@
     System health monitor daemon for SONiC
 """
 
+from swsscommon.swsscommon import RestartWaiter
+RestartWaiter.waitAdvancedBootDone()
```

Comment:
Does this call need to be at this point? Can it be in the main method?

Reply:
@saiarcot895 no, we prefer to wait before application starts importing python modules since importing some can be pretty CPU expensive as some run code at import time.